docs/src/Sheet.doc.js

```diff
@@ -698,6 +705,60 @@ function RefExample() {
   />
 );
 
+card(
+  <Example
+    id="shouldCloseOnKeyEventExample"
+    name="Example: shouldCloseOnKeyEvent"
+    description={`
+    A \`Sheet\` with shouldCloseOnKeyEvent
```

Comment:
Could you elaborate a bit more here as to when people would want this feature disabled, and maybe add a tooltip/flyout to the sheet to help demonstrate the behavior. 

Maybe something like "The `shouldCloseOnKeyEvent` may need to be disabled if the Sheet contains components that rely on the escape key themselves. When this property is disabled, using the escape key to close an inner component will not inadvertently close the Sheet. If all inner components are closed, this property should be restored to true to allow for the escape key to close the Sheet."

Reply:
Done